ci/test-elk-flow-collector.sh

```diff
@@ -4,18 +4,27 @@ LOGSTASH_PORT=4739
 LOGSTASH_IP="0.0.0.0"
 LOGSTASH_PROTOCOL="udp"
 FAILURE=false
+export KUBECONFIG=${WORKSPACE}/jenkins/out/kubeconfig
+
+start_antrea() {
+  echo "=== Starting Antrea ==="
+  kubectl apply -f ${WORKSPACE}/build/yamls/antrea.yml
+  kubectl rollout status --timeout=5m deployment/coredns -n kube-system
+  kubectl rollout status --timeout=5m deployment.apps/antrea-controller -n kube-system
+  kubectl rollout status --timeout=5m daemonset/antrea-agent -n kube-system
+}
 
 setup_flow_collector() {
   echo "=== Starting Flow Collector ==="
-  kubectl create namespace antrea-flow-collector
-  kubectl create configmap logstash-configmap -n antrea-flow-collector --from-file=${GIT_CHECKOUT_DIR}/build/yamls/flow-collector/logstash/
-  kubectl apply -f ${GIT_CHECKOUT_DIR}/build/yamls/flow-collector/flow-collector.yml -n antrea-flow-collector
+  kubectl create namespace elk-flow-collector
+  kubectl create configmap logstash-configmap -n elk-flow-collector --from-file=${WORKSPACE}/build/yamls/elk-flow-collector/logstash/
+  kubectl apply -f ${WORKSPACE}/build/yamls/elk-flow-collector/elk-flow-collector.yml -n elk-flow-collector
   echo "=== Waiting for Elastic Stack to be ready ==="
-  kubectl wait --for=condition=ready pod -l app=elasticsearch -n antrea-flow-collector --timeout=600s
-  kubectl wait --for=condition=ready pod -l app=kibana -n antrea-flow-collector
-  kubectl wait --for=condition=ready pod -l app=logstash -n antrea-flow-collector --timeout=600s
+  kubectl wait --for=condition=ready pod -l app=kibana -n elk-flow-collector
```

Comment:
out-of-curiosity why is this the only one for whcih we don't have a `--timeout` argument

Reply:
Kibana is usually ready right away, so default 30s timeout should be enough.

Reply:
Actually I just tested and it failed. Maybe add 10s timeout.